meta/InlineConfigParser.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace dokuwiki\plugin\struct\meta;
+
+/**
+ * Class InlineConfigParser
+ *
+ * Wrapper to convert inline syntax to full before instantiating ConfigParser
+ *
+ * @package dokuwiki\plugin\struct\meta
+ */
+class InlineConfigParser extends ConfigParser {
+
+    /**
+     * Parser constructor.
+     *
+     * parses the given inline configuration
+     *
+     * @param  string  $inline
+     */
+    public function __construct($inline) {
+        // To support a fuller syntax, such as user-specified filters, tokenise here
+
+        // Split into components
+        $components = str_getcsv($inline, '.');
```

Comment:
Cool idea to use the CSV parsing here to support quoting and stuff, but I wonder if that's necessary. Wouldn't a simple `explode` be sufficient?

Reply:
That was my first thought, but dots are allowed in page names, so some more intelligent handling needs to be used. Rather than rewriting the logic, I decided to use the built-in.

Reply:
Oh, reading on to the other comments, this can be dealt with another way...